tests/test-run.sh

```diff
@@ -50,15 +50,28 @@ if ${is_uidzero} || test -x `dirname $UNREADABLE`; then
 fi
 
 # https://github.com/projectatomic/bubblewrap/issues/217
-BWRAP_RO_HOST_ARGS="--ro-bind /usr /usr
-          --ro-bind /etc /etc
-          --dir /var/tmp
-          --symlink usr/lib /lib
-          --symlink usr/lib64 /lib64
-          --symlink usr/bin /bin
-          --symlink usr/sbin /sbin
-          --proc /proc
-          --dev /dev"
+# are we on a merged-/usr system?
+if [ -L /lib ]; then
```

Comment:
Maybe `[ /lib -ef /usr/lib ]` would be better? That checks whether they are the same file after dereferencing symlinks.

Or maybe just use the unmerged-`/usr` style everywhere.

Reply:
I'll do the first part, that makes sense.

Happy to do the second if that's the consensus.